app/_locales/en/messages.json

```diff
@@ -44,6 +44,57 @@
   "QRHardwareWalletSteps2Description": {
     "message": "Ngrave (Coming Soon)"
   },
+  "SIWEAddressInvalid": {
+    "message": "The address in the sign-in request does not match the address of the account you are using to sign in."
+  },
+  "SIWEDomainInvalid": {
+    "message": "The website you are attempting to sign in to ($1) does not match the domain in the sign-in request. Proceed with caution."
+  },
+  "SIWEDomainWarningBody": {
+    "message": "The website ($1) is asking you to sign in to the wrong domain. This may be a phishing attack."
```

Comment:
```suggestion
    "message": "The website ($1) is asking you to sign in to the wrong domain. This may be a phishing attack.",
    "description": "$1 represents the website domain"
```